resources/views/layouts/components/head.blade.php

```diff
@@ -51,4 +51,4 @@
     {{-- END v1 compat --}}
 
     @stack('head')
-</head>
+</head>
```

Comment:
nit: Whatever changed here can probably be reverted so it doesn't make it into this PR's changeset.